designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputCheckBox.kt

```diff
@@ -85,6 +87,8 @@ fun InputCheckBox(
                         onClearSelection.invoke()
                     },
                 )
+            } else {
+                Spacer(modifier = Modifier.width(Spacing.Spacing24))
             }
```

Comment:
@DavidAparicioAlbaAsenjo This component still change it's space. I think we need to update the Size to 48 as the size of icon is 48 too. 
<img width="781" alt="image" src="https://github.com/dhis2/dhis2-mobile-ui/assets/25721637/970ae57f-a1cf-49f9-b2dc-121b9a71929f">
<img width="779" alt="image" src="https://github.com/dhis2/dhis2-mobile-ui/assets/25721637/916043c2-3946-4b70-85d2-cd0ed5439056">